k8s/apis/k8s/v1alpha1/webhook_test.go

```diff
@@ -47,19 +49,13 @@ var (
 	cancel    context.CancelFunc
 )
 
-func TestAPIs(t *testing.T) {
-	RegisterFailHandler(Fail)
-
-	RunSpecs(t,
-		"Webhook Suite")
-}
-
-var _ = BeforeSuite(func() {
+func Test_BeforeSuite(t *testing.T) {
```

Comment:
I think you'd better to under how the [ginkgo framework ](https://onsi.github.io/ginkgo/)works. BeforeSuite is used to build the environment for the test suit, you can't update it as a single test function. Also, please check the other similar parts.